core/dbt/logger.py

```diff
@@ -20,20 +20,19 @@
 # then we should override the logging stream to use the colorama
 # converter. If the TERM var is set (as with Git Bash), then it's safe
 # to send escape characters and no log handler injection is needed.
-colorama_stdout = sys.stdout
-colorama_wrap = True
+logging_stdout = sys.stdout
+if sys.platform == "win32":
+    colorama_wrap = True
 
-colorama.init(wrap=colorama_wrap)
+    colorama.init(wrap=colorama_wrap)
 
+    if not os.getenv("TERM"):
+        colorama_wrap = False
+        logging_stdout = colorama.AnsiToWin32(sys.stdout).stream
+    else:
+        colorama_wrap = False
 
-if sys.platform == "win32" and not os.getenv("TERM"):
-    colorama_wrap = False
-    colorama_stdout = colorama.AnsiToWin32(sys.stdout).stream
-
-elif sys.platform == "win32":
-    colorama_wrap = False
-
-colorama.init(wrap=colorama_wrap)
+    colorama.init(wrap=colorama_wrap)
 
```

Comment:
This section is kind of hard to read before your change. I tried to reformat this section with the changes made, the logic is being updated to 
```
if sys.platform == "win32" and not os.getenv("TERM"):
    logging_stdout = colorama.AnsiToWin32(sys.stdout).stream
else:
    logging_stdout = sys.stdout

if sys.platform == "win32":
    colorama_wrap = False
    colorama.init(wrap=colorama_wrap)
```
Previous logic after similar refactor looks like
```
if sys.platform == "win32" and not os.getenv("TERM"):
    colorama_stdout = colorama.AnsiToWin32(sys.stdout).stream
else:
    colorama_stdout = sys.stdout

if sys.platform == "win32":
    colorama_wrap = False
else:
    colorama_wrap = True
colorama.init(wrap=colorama_wrap)
```
So seems like what you did was remove the initialization of `colorama ` when not running on windows. Is that what we wanted to do?

Reply:
Yes, my intent was to remove the initialization of `colarama` when not running on Windows as I noticed that seemed to resolve the issue for me when running on Mac and Linux

Regarding your suggested changes, yes that looks okay to me. I can try and test it myself locally to verify.

Reply:
Ideally we'd fix the issue on Windows. My solution here doesn't resolve the issue on Windows when I tested it.

But I'm not too familiar with Windows console or colorama to suggest any solutions at the moment. Any insight from the dbt dev team would be useful if possible.

Reply:
I personally don't have much experience with windows either. We can leave the windows part out!